dev/portfolio/SQL/SQL.md

```diff
@@ -0,0 +1,21 @@
+## MySQL in Node.js
```

Comment:
Great. This closes #14 - See [this article](https://bloggie.io/@kinopyo/github-tip-closing-issue-automatically-when-the-pull-request-is-merged) on how to note completed issues in PRs.